<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="maze_wall">
    <static>true</static>
    <link name="link">
      <pose>0 0 0 0 0 0</pose>
      <inertial>
        <mass>1000</mass>
        <inertia>
          <ixx>0.0008611666666666669</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.0008611666666666669</iyy>
          <iyz>0</iyz>
          <izz>0.0008611666666666669</izz>
        </inertia>
      </inertial>

      <collision name="collision">
        <geometry>
          <box>
            <size>.18 .012 .050</size>
            <!--Note that the wall length is set to (.168 + .012) so that there are no "gaps" between wall pieces. This may cause some walls to protrude ~.012 passed where they should, but this may actually be how the maze is set up -->
          </box>
        </geometry>
        <surface>
          <contact>
            <!-- Red Pine coefficients for longitudinal axis of the wood
                 according to:
                 http://www.fpl.fs.fed.us/documnts/fplgtr/fplgtr113/ch04.pdf -->
            <poissons_ratio>0.347</poissons_ratio>
            <elastic_modulus>8.8e+09</elastic_modulus>
            <ode>
              <kp>1000000</kp>
              <kd>100</kd>
              <max_vel>100.0</max_vel>
              <min_depth>0.0001</min_depth>
            </ode>
          </contact>
          <friction>
            <torsional>
              <coefficient>1.0</coefficient>
              <use_patch_radius>0</use_patch_radius>
              <surface_radius>0.01</surface_radius>
            </torsional>
          </friction>
        </surface>
      </collision>

      <visual name="visual">
        <geometry>
          <box>
            <size>.18 .012 .080</size>
          </box>
        </geometry>
        <material>
          <script>
            <uri>file://media/materials/scripts/gazebo.material</uri>
            <name>Gazebo/Wood</name>
          </script>
        </material>
      </visual>

    </link>
  </model>
</sdf>
